notebooks/02_model/fm_deep_dive.ipynb

```diff
@@ -0,0 +1,1530 @@
+{
```

Comment:
<p>do you mind to add a cell with the parameters, so we can inject them from papermill?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Ah yea. Will do